cmdx.py

```diff
@@ -5258,6 +5258,7 @@ class Compound(_AbstractAttribute):
     Multi = None
 
     def __init__(self, name, children=None, **kwargs):
+        self.Fn = om.MFnCompoundAttribute()  # see https://github.com/mottosso/cmdx/issues/5#issuecomment-717330454
```

Comment:
Should we make the class `Fn = None`? Of of these aren't necessary.

Also try and stick to 80 lines with if possible, which it can be here if we put the comment on the line above.

Reply:
Good point, removed.

Actually I was wondering if we should keep declaring `Fn` as currently (static for each attribute type), but always instantiate something like `self.Fn = cls.Fn()` directly in `_AbstractAttribute.__init__()`, but I'm not 100% sure this is strictly necessary outside for non-Compound attributes.
Also I'm not sure how to actually get to the type-specific `Fn` from `_AbstractAttribute` 😄